src/mutations/itemExport.ts

```diff
@@ -20,6 +20,10 @@ export default (queryConfig: QueryClientConfig) => {
     });
 
   return {
-    useExportZip,
+    useExportItem,
+    /**
+     * @deprecated use useExportItem
+     */
+    useExportZip: useExportItem,
```

Comment:
Actually the change is breaking anyways no ?

Reply:
it was more for the rename, but true it will be breaking anyway, so I'll just rename.